src/lib/icu/meson.build

```
@@ -18,17 +18,26 @@ if icu_dep.found()
     'Util.cxx',
     'Init.cxx',
   ]
-elif not get_option('iconv').disabled()
-  # an installed iconv library will make the builtin iconv() unavailable,
-  # so search for the library first and pass it as (possible) dependency
-  iconv_dep = compiler.find_library('libiconv', required: false)
-  have_iconv = compiler.has_function('iconv', 
-    dependencies: iconv_dep, 
-    prefix : '#include <iconv.h>')
-  if not have_iconv and get_option('iconv').enabled()
-    error('iconv() not available')
+else
+  if meson.version().version_compare('>= 0.60')
+    iconv_dep = dependency('iconv', required: get_option('iconv'))
+    have_iconv = iconv_dep.found()
+    conf.set('HAVE_ICONV', have_iconv)
```

have_iconv is only used for conf.set, and for older meson versions to track whether another try should be done. For the 0.60.0 case, you don't need to set this bool variable at all, so you can just do:

```suggestion
    conf.set('HAVE_ICONV', iconv_dep.found())
```

---

Yeah. It''s from when I tried to keep changes to a minimum.